pkg/idtools/idtools_windows.go

```diff
@@ -1,24 +1,13 @@
-package idtools // import "github.com/docker/docker/pkg/idtools"
+package idtools
 
 import (
 	"os"
-
-	"github.com/docker/docker/pkg/system"
-)
-
-const (
-	SeTakeOwnershipPrivilege = "SeTakeOwnershipPrivilege"
-)
-
-const (
-	ContainerAdministratorSidString = "S-1-5-93-2-1"
-	ContainerUserSidString          = "S-1-5-93-2-2"
 )
 
 // This is currently a wrapper around MkdirAll, however, since currently
 // permissions aren't set through this path, the identity isn't utilized.
 // Ownership is handled elsewhere, but in the future could be support here
 // too.
 func mkdirAs(path string, _ os.FileMode, _ Identity, _, _ bool) error {
-	return system.MkdirAll(path, 0)
+	return os.MkdirAll(path, 0)
 }
```

Comment:
I went looking if the `system.MkdirAll` was needed at all anywhere (for the special handling of windows GUID volume paths), and it looks like it's no longer needed since go1.22, so I opened a PR to deprecate it, and remove its use;

- https://github.com/moby/moby/pull/49162


☝️ this was the last use go `pkg/system` in BuildKit, so removing this line makes BuildKit no longer depend on it.

Reply:
Did a quick rebase after that one was merged